docs/content/toc.textile

```diff
@@ -16,6 +16,7 @@ h2. Data Ingestion
 * "Schema Design":../ingestion/schema-design.html
 * "Realtime Ingestion":../ingestion/realtime-ingestion.html
 * "Batch Ingestion":../ingestion/batch-ingestion.html
+* "Recompressing Segments":../ingestion/convert.html
```

Comment:
I think `Converting Segments` would be more appropriate, since it does other things than just re-compress.